test/RawDataset.js

```diff
@@ -278,6 +278,32 @@ describe("RawDataset: Raw Datasets", () => {
     );
   });
 
+  it("should update comment of the dataset", async () => {
+    return request(appUrl)
+      .patch("/api/v3/datasets/" + pid)
+      .send(TestData.PatchComment)
+      .set("Accept", "application/json")
+      .set({ Authorization: `Bearer ${accessTokenArchiveManager}` })
+      .expect(200)
+      .expect("Content-Type", /json/)
+      .then((res) => {
+        res.body.should.have.property("comment").and.be.string;
+      });
+  })
+
+  it("should update data quality metrics of the dataset", async () => {
```

Comment:
Can we add also a test updating quality metrics with a string (or an invalid value) which should fail?